### ISSUE TITLE
NFTs Royalty Pallet for Polkadot Asset Hub

### ISSUE THREAD

**Issue by yangwao** [kodadot/nft-gallery#6971]
@niklasp have you seen this one?
- https://forum.polkadot.network/t/nfts-royalty-pallet/3766

ref
---
- https://github.com/kodadot/nft-gallery/issues/6970

**Comment by niklasp** [comment]
Not looked in the code yet but were discussing with @vikiival and Jegor from parity about the multi royalty format and the one were using seems to have standards potential.

Would be nice to have a better display here,

E.g.

Receiver 1: x%
Receiver 2: y%

**Comment by roiLeo** [comment]
any news? will we have on substrate nft royalties pallet?

**Comment by vikiival** [comment]
@jsidorenko

**Comment by jsidorenko** [comment]
nope, I doubt there will be a royalties pallet on AH. On the other hand, we're exploring ways to extend the AssetHub NFTs functionality via smart contracts, in that case, royalties could become part of the contract logic
